lms/djangoapps/verify_student/services.py

```diff
@@ -218,6 +219,7 @@ def user_status(cls, user):
             expiration_datetime = cls.get_expiration_datetime(user, ['approved'])
             if getattr(attempt, 'expiry_date', None) and is_verification_expiring_soon(expiration_datetime):
                 user_status['verification_expiry'] = attempt.expiry_date.date().strftime("%m/%d/%Y")
+            user_status['status_date'] = attempt.updated_at
```

Comment:
I would say rather than use the `updated_at` to inform the `status_date`, you should use the `status_changed` of the attempt.